ccv/management/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since we're developing in Python 3 and UTF-8 is by default I don't find these comments to be useful. According to https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3 the one reason remaining for these lines is for some IDE so if yours need those, then I guess it's fine, but otherwise (if you use pycharms or vim) I would get rid of these lines, all over the project.

Reply:
I've removed the UTF-8 comments from all over the project.